src/Apps/Search/Routes/Artworks/Components/Filter/SearchResultsRefetch.tsx

```diff
@@ -54,6 +57,25 @@ export class SearchResultsRefetch extends Component<SearchRefetchProps> {
             console.error(error)
           }
 
+          const term = this.props.filtersState.keyword
+          const filters = Object.assign({}, this.props.filtersState)
+
+          const removeEmptyOrDefaults = obj => {
+            Object.keys(obj).forEach(
+              key => isDefaultFilter(key, obj[key]) && delete obj[key]
+            )
+          }
+          removeEmptyOrDefaults(filters)
+
+          const filterQueryParams = qs.stringify({
+            ...omit(filters, "keyword"),
+            term,
+          })
```

Comment:
This block can probably be simplified to 
```js
const filters = Object.entries(this.props.filtersState).reduce((acc, [key, value]) => {
  if (isDefaultFilter(key, value) || key === 'keyword') {
     return acc
  } else {
    ...acc,
    [key]: value
  }
}, {}) 

const filterQueryParams = qs.stringify({ ...filters, term })
```

Which also removes some of the weirdness around mutation.

Reply:
oo i like!